backend/app/helpers/spree/admin/navigation_helper.rb

```diff
@@ -31,6 +31,7 @@ def tab(*args, &_block)
         elsif options[:match_path]
           request.fullpath.starts_with?("#{admin_path}#{options[:match_path]}")
         else
+          request.fullpath.starts_with?(destination_url) ||
           args.include?(controller.controller_name.to_sym)
```

Comment:
Could we have an indent here to show that this is part of the previous statement?